src/data/fintype/basic.lean

```diff
@@ -519,6 +532,27 @@ instance Prop.fintype : fintype Prop :=
 def set_fintype {α} [fintype α] (s : set α) [decidable_pred s] : fintype s :=
 fintype.subtype (univ.filter (∈ s)) (by simp)
 
+namespace function.embedding
+
+/-- An embedding from a `fintype` to itself can be lifted to an equivalence. -/
```

Comment:
```suggestion
/-- An embedding from a `fintype` to itself can be promoted to an equivalence. -/
```

Reply:
"lifted" isn't wrong, I suppose, but maybe some variety in language would be nice.